terraform/modules.tf.json.template.py

```diff
@@ -1,6 +1,14 @@
 from azul import config
 from azul.deployment import emit_tf
 
+
+def get_es_domain():
+    if not config.share_es_domain:
```

Comment:
Kind of a nit, but could you remove the `not` and switch the order?

Reply:
Also I think Hannes prefers methods that don't start with `get`.

Finally, consider (very optional) inlining this as an expression.